apps/test-bundles/.gitignore

```diff
@@ -0,0 +1 @@
+fixtures/*
```

Comment:
It would probably be preferable to put temporary files in the OS temp directory (see `packages/cra-template/scripts/test.ts` for an example). Or possibly in a `temp` folder within the current package, since that should already be ignored.

Reply:
I still think the fixtures should at least be moved underneath `temp/fixtures` (within the current package directory) so that they'll be automatically ignored, both by git and typically by other tools too.

Reply:
Moved to temp/fixtures and removed .gitignore file. Thanks for the suggestion!